### PR TITLE
Improved accessibility and readability to comply with WCAG

### PR DESCRIPTION
**Pull request by robinpiets**
I've added some quick fixes to improve accessibility and readability according to a WCAG quick scan

### CODE REVIEW

**Review by svenjens — APPROVED** [review]
Nice improvements! @svenjens approves!

File: packages/client/src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ const FirstColumn = () => (
         <PhoneNumber
           darkBackground
           eventName={sections.FOOTER}
+          underline
```
Comment (afjlambert) [inline packages/client/src/components/Footer/Footer.tsx:32]: underline should not be a property of PhoneNumber imho. If you want to render the phone number in print for example, it does not make sense to underline. Underline is a property of a link, not a piece of text.

File: packages/client/src/components/HeaderStyles.ts [hunk 2]
```diff
@@ -46,6 +46,7 @@ export const StyledLogo = styled.span`
   cursor: pointer;
   background-size: cover;
   background-image: url("${LogoMobile}");
+  text-indent: -9999px;
```
Comment (afjlambert) [inline packages/client/src/components/HeaderStyles.ts]: Is this related to wag? Maybe add a comment if so.

File: packages/client/src/components/Link.tsx [hunk 3]
```diff
@@ -23,7 +15,7 @@ const Link = ({
   eventName,
   href,
   ...rest
-}: LinkProps) => {
+}: LinkProps & ComponentProps<typeof StyledComponentLink>) => {
```
Comment (afjlambert) [inline packages/client/src/components/Link.tsx]: Much improve 👍

File: packages/client/src/components/PhoneNumber.tsx [hunk 4]
```diff
@@ -1,12 +1,20 @@
...
 import { PHONE_NUMBER } from "../utils/test-ids";
 import Link from "./Link";
 
-const Wrapper = styled.span`
+type WrapperProps = { underline?: boolean };
+
+const Wrapper = styled.span<WrapperProps>`
   white-space: nowrap;
+
+  ${({ underline }) =>
```
Comment (afjlambert) [inline packages/client/src/components/PhoneNumber.tsx]: Me like syntax 🙌

**Review by afjlambert — APPROVED** [review]
Small stuff, maybe change the underline on PhoneNumber?

File: packages/client/src/components/Footer/Footer.tsx (hunk 1 above)
Comment (robinpiets) [reply packages/client/src/components/Footer/Footer.tsx:32]: I understand what you mean, I will place the underline inside the Link. The PhoneNumber, however, still needs an underline prop
